packages/getlit-sdk/src/lib/ipfs-provider-sdk/providers/BaseIPFSProvider.ts

```diff
@@ -0,0 +1,15 @@
+import { CID } from 'blockstore-core/dist/src/base';
+
+export abstract class BaseIPFSProvider {
+  name: string;
+
+  constructor(name: string) {
+    this.name = name;
+  }
+
```

Comment:
We should add some `initalize` to the interface for bootstrapping dependencies. we can move what is in `createHeliaNode` here